packages/techdocs-cli/package.json

```diff
@@ -63,7 +63,7 @@
     "@backstage/catalog-model": "^0.10.0",
     "@backstage/cli-common": "^0.1.7",
     "@backstage/config": "^0.1.14",
-    "@backstage/techdocs-common": "^0.11.8",
+    "@backstage/techdocs-common": "^0.11.10",
```

Comment:
@camilaibs  I _believe_ you don't have to manually bump this version in the PR.  With the changeset you wrote here, it should automatically bump the version of `@backstage/techdocs-common` during the next release.

Reply:
It's true, removed.